torch/ao/quantization/fx/node_check_utils.py

```diff
@@ -0,0 +1,37 @@
+import torch
+
+module_type_list = {
+    torch.nn.ReLU,
+    torch.nn.ReLU6,
+    torch.nn.AdaptiveAvgPool1d,
+    torch.nn.AdaptiveAvgPool2d,
+    torch.nn.AdaptiveAvgPool3d,
+    torch.nn.AvgPool1d,
+    torch.nn.AvgPool2d,
+    torch.nn.AvgPool3d,
+    torch.nn.MaxPool1d,
+    torch.nn.MaxPool2d,
+    torch.nn.MaxPool3d,
+}
+func_list = {
+    torch.nn.functional.adaptive_avg_pool1d,
+    torch.nn.functional.adaptive_avg_pool2d,
+    torch.nn.functional.adaptive_avg_pool3d,
+    torch.nn.functional.max_pool1d,
+    torch.nn.functional.max_pool2d,
+    torch.nn.functional.max_pool3d,
+    torch.nn.functional.relu,
+    torch.nn.functional.hardtanh,
+    torch.nn.functional.hardtanh_,
+}
+method_list = {
+    torch.mean,
+    'relu',
+    'relu_',
+}
+
+def chech_qnode_func(node, modules):
```

Comment:
probably a typo? chech -> check?

Reply:
I think it would be better to rename this to `is_flow_supported_ops` and rename `is_call_function` to `is_supported_call_function` and same for `is_call_method` and `is_call_module` to be clearer